openedx.yaml

```diff
@@ -0,0 +1,4 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oep-0002.html#specification
+
+archived: True
```

Comment:
I am adding the openedx.yaml file according to the modifications to OEP-2 proposed in https://github.com/edx/open-edx-proposals/pull/40.